prime-router/src/main/kotlin/history/ReportHistory.kt

```diff
@@ -37,6 +38,7 @@ abstract class ReportHistory(
     var reportId: String? = null,
     var topic: Topic? = null,
     var reportItemCount: Int? = null,
+    var receivingOrgSvcStatus: String? = null,
```

Comment:
Now that we've split up the DB actions, maybe makes sense to only do this for the a Delivery?

I see this
```
{
        "submissionId": 427,
        "timestamp": "2024-03-13T12:16:54.826Z",
        "id": "cb7451ff-929a-4633-ae51-658fdc51ca9d",
        "topic": "elr-elims",
        "reportItemCount": 1,
        "httpStatus": 201,
        "receivingOrgSvcStatus": null,
        "sender": "ignore.ignore-elr-elims"
    },
```
locally when using the submissions API